prometheus-ksonnet/lib/config.libsonnet

```diff
@@ -38,6 +38,7 @@
     prometheus_path: '/prometheus/',
     prometheus_port: 80,
     prometheus_web_route_prefix: $._config.prometheus_path,
+    prometheus_config_dir: '/etc/prometheus',
```

Comment:
This only changes the path that prometheus looks in for the config, but does not change the path that the configMap is mounted into.  I understand this is the intent here, but i feel users will see the config option and not understand the impact of changing it.

can we not just override the prometheus container args or configMap mount path using jsonnet?

Reply:
Agreed, this is a bit confusing, reverting that.